section2/exercises/if-statements.js

```diff
@@ -82,14 +91,21 @@ When cupsOfFlour = 2 and hasSauce = true, your program should log "I can make pi
 When cupsOfFlour = 3 and hasSauce = true, your program should log "I can make pizza";
 */
 
+// Write your conditional here
+
 var cupsOfFlour = 1;
 var hasSauce = true;
 
-// Write your conditional here
+if (cupsOfFlour >= 2 && hasSauce === true) {
```

Comment:
Great solution!

Get that `else` up on line 101 right after the `}` for style and syntax conventions.